cmd/troubleshoot.go

```diff
@@ -30,7 +29,7 @@ If you're running into trouble, copy and paste the output from the troubleshoot
 command into a GitHub issue so we can help figure out what's going on.
 `,
 	RunE: func(cmd *cobra.Command, args []string) error {
-		cli.HTTPClient = &http.Client{Timeout: 20 * time.Second}
+		cli.TimeoutInSeconds = 20
```

Comment:
What if we multiply the default value as opposed to hard coding to 20s?  If a user is having a problem pining the API and setting a timeout it would appear that the timeout is not enough, but in fact it is always 20s which the user may not know or care to look at the source.

Reply:
You mean, in troubleshoot say something like `cli.TimeoutInSeconds = cli.TimeoutInSeconds * 5` or something?

Reply:
Let's start with 2 and see how we fare :)